membership_file/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Please add those.

I'm especially interested in the test for the deletion logic.